.circleci/config.yml

```diff
@@ -1094,6 +1132,7 @@ jobs:
     environment:
       <<: *base_osx_env
       CMAKE_BUILD_TYPE: Release
+      CMAKE_OPTIONS: -DCMAKE_OSX_ARCHITECTURES:STRING=x86_64;arm64
```

Comment:
That's quite cool - if this is all that's needed, then we don't even need a headache over https://github.com/orgs/Homebrew/discussions/412 and homebrew accepting universal binaries or not, since the builds there can just set it as they want from the outside like this